### PR TITLE
Query : Start & Stop Baker

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose

- Added support for query `StartBaker`
- Added support for query `StopBaker`

## Changes

- Added support for query `StartBaker`
- Added support for query `StopBaker`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,18 @@ Retrives the node software version.
 NodeInfo getNodeInfo()
 ```
 
+- `startBaker`
+```java
+boolean startBaker()
+```
+Start the baker in the consensus module.
```
Comment (EmilLa1) [inline README.md:227]:
```suggestion
Start the baker.
```

File: README.md [hunk 2]
```diff
@@ -220,6 +220,18 @@ Retrives the node software version.
...
+
+- `stopBaker`
+```java
+boolean stopBaker()
+```
+Stop the baker in the consensus module.
```
Comment (EmilLa1) [inline README.md:233]:
```suggestion
Stop the baker.
```

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java [hunk 3]
```diff
@@ -325,6 +322,24 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Start the baker in the consensus module.
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/Client.java:326]:
```suggestion
     * Start the baker.
```

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java [hunk 4]
```diff
@@ -325,6 +322,24 @@ public Semver getVersion() {
...
+     *
+     * @return true if baker could be started. false otherwise.
+     */
+    public boolean startBaker() {
+        return server().startBaker(ConcordiumP2PRpc.Empty.newBuilder().build()).getValue();
+    }
+
+    /**
+     * Stop the baker in the consensus module.
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/Client.java:335]:
```suggestion
     * Stop the baker.
```

**Review by EmilLa1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (parv0888) [reply README.md:227]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java (hunk 3 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/Client.java:326]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java (hunk 4 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/Client.java:335]: done

**Review by EmilLa1 — APPROVED** [review]

**Pull request merged**